contrib/intel/jenkins/runtests.py

```diff
@@ -35,13 +35,16 @@ def __call__(self, parser, namespace, values, option_string=None):
                     "variables", nargs='*', action=ParseDict, default={})
 parser.add_argument('--mpi', help="Select mpi to use for middlewares",
                     choices=['impi', 'mpich', 'ompi'], default='impi')
+parser.add_argument('--log', help="Full path to log file",
+                    default=os.environ['DEFAULT_LOG_LOCATION'], type=str)
 
 args = parser.parse_args()
 args_core = args.prov
 
 args_util = args.util
 args_device = args.device
 user_env = args.user_env
+log = args.log
 
```

Comment:
nit: why not just call it log file. :)

Reply:
Done! There was no reason I called it log vs log_file